sdk/storage/azfile/file/examples_test.go

```diff
@@ -0,0 +1,281 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package file_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/streaming"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/file"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/service"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azfile/share"
+	"io"
+	"log"
+	"os"
+	"strings"
+)
+
+func handleError(err error) {
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+}
+
+const random64BString string = "2SDgZj6RkKYzJpu04sweQek4uWHO8ndPnYlZ0tnFS61hjnFZ5IkvIGGY44eKABov"
+
+func generateData(sizeInBytes int) (io.ReadSeekCloser, []byte) {
+	data := make([]byte, sizeInBytes)
+	_len := len(random64BString)
+	if sizeInBytes > _len {
+		count := sizeInBytes / _len
+		if sizeInBytes%_len != 0 {
+			count = count + 1
+		}
+		copy(data[:], strings.Repeat(random64BString, count))
```

Comment:
instead of repeating a static string for each test can we generate random data based on a seed (that is generated from the test name)? That way we are testing different data for each test. There should be an example in blobs of this.

Reply:
But I guess these are just samples, maybe it doesnt matter?

Reply:
it samples, so hardcoded size is fine, I can however make that change for tests